source/Kepler-System-Tests/SystemHotInstallerTest.class.st

```diff
@@ -0,0 +1,26 @@
+"
+I'm a Test Case for SystemHotInstaller
+"
+Class {
+	#name : #SystemHotInstallerTest,
+	#superclass : #TestCase,
+	#category : #'Kepler-System-Tests'
+}
+
+{ #category : #tests }
+SystemHotInstallerTest >> testInstalling [
+
+	| composite cms |
+
+	composite := CompositeSystem new.
+	composite startUp.
+	self should: [ composite systemImplementing: #CustomerManagementSystem ] raise: SystemControlError.
```

Comment:
Shouldn't we assert about the error message returned as well?

Reply:
I don't think so. I tested that in another place, here the important thing is that there's no system implementing this interface before we perform the hot install.